src/cdi/CDI_integratePlanckSpectrum.cc

```diff
@@ -32,8 +32,9 @@ double CDI::integratePlanckSpectrum(double low, double high, const double T) {
   Require(high >= low);
   Require(T >= 0.0);
 
-  // return 0 if temperature is a hard zero
-  if (rtt_dsxx::soft_equiv(T, 0.0, std::numeric_limits<double>::epsilon()))
+  // high/T must be < numeric_limits<double>::max().  So, if T ~< high*min, then
+  // return early with zero values (assuming max() ~ 1/min()).
+  if (T < high * std::numeric_limits<double>::min())
```

Comment:
This is a very nice way to do the check!
Is there any motivation to use `std::numeric_limits<double>::epsilon()`, since it has now been non-dimensionalized?